cityofaustin/atd-moped
# Tear down activity log pipeline and clean up unused code

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10897

This is a lot of deleting. I removed the moped-events folder and the `/events` route from the moped-api app. I also removed the github issue templates and labels since issues should be opened in the atd-data-tech repo. 

I left a note asking if we should completely remove the `atd_moped_build_stage.yml` workflow, the workflow doesnt work after https://github.com/cityofaustin/atd-moped/pull/781/ this was merged. 

After this is merged to main I can remove the staging assets that are in AWS (sqs queue, lambda function). And the production ones can be removed after we do the next release. 


## Testing
**URL to test:** 

https://deploy-preview-975--atd-moped-main.netlify.app/moped/projects/1862?tab=activity_log

**Steps to test:**

Check to see that the activity log renders without issue, including events before last november, after removing all the old activity log code. 

---
#### Ship list
